code/_onclick/hud/screen_objects.dm

```diff
@@ -453,8 +453,9 @@
 				. += handcuff_overlay
 
 			var/obj/item/organ/external/hand = C.get_organ("[slot_id == slot_l_hand ? "l" : "r"]_hand")
-			if(!hand || !hand.is_usable())
-				. += blocked_overlay
+			if(!isalien(C))
+				if(!hand || !hand.is_usable())
+					. += blocked_overlay
```

Comment:
```suggestion
			if(!isalien(C) && !hand || !hand.is_usable())
				. += blocked_overlay
```

Reply:
I think those need parentheses around the `!hand || !hand.is_usable())`